.env.example

```diff
@@ -1,10 +0,0 @@
-# rename this file to .env and supply the values listed below
-# also make sure they are available to the build tool (e.g. Netlify)
-# warning: variables prefixed with GATSBY_ will be made available to client-side code
-# be careful not to expose sensitive data (e.g. your Algolia admin key)
-ALGOLIA_ADMIN_KEY=insertValue
-DEFI_PULSE_API_KEY=insertValue
-ETHERSCAN_API_KEY=insertValue
-GATSBY_ALGOLIA_APP_ID=insertValue
-GATSBY_ALGOLIA_SEARCH_KEY=insertValue
-GATSBY_GITHUB_TOKEN_READ_ONLY=insertValue
```

Comment:
Careful here 😃  We don't want this deleted, mind reverting this?

Reply:
@wackerow reverted the .env file 😊. Thanks .